modules/models/src/main/java/org/eclipse/xpanse/modules/models/billing/RatingMode.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.billing;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.io.Serial;
+import java.io.Serializable;
+import lombok.Data;
+
+/**
+ * Defines the rating mode of the managed service.
+ */
+@Data
+public class RatingMode implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 240913796673011260L;
+
+    @Schema(description = "The fixed price of the flavor in the managed service.")
```

Comment:
Extend the comment as - The fixed price of the flavor in the managed service. This price includes all prices and this is the price shown to the customer.